eng/common/templates/stages/build-test-publish-repo.yml

```diff
@@ -46,24 +46,24 @@ stages:
 - stage: Build
   condition: and(succeeded(), contains(variables['stages'], 'build'))
   jobs:
-  - template: ../jobs/test-images-linux-client.yml
-    parameters:
-      name: PreBuildValidation
-      pool: ${{ parameters.linuxAmd64Pool }}
-      testJobTimeout: ${{ parameters.linuxAmdTestJobTimeout }}
-      preBuildValidation: true
-      internalProjectName: ${{ parameters.internalProjectName }}
-      customInitSteps:
-        - ${{ parameters.customTestInitSteps }}
-        # These variables are normally set by the matrix. Since this test job is not generated
-        # by a matrix, we need to set them manually. They can be set to empty values since their
-        # values aren't actually used for the pre-build tests.
-        - powershell: |
-            echo "##vso[task.setvariable variable=productVersion]"
-            echo "##vso[task.setvariable variable=imageBuilderPaths]"
-            echo "##vso[task.setvariable variable=osVersions]"
-            echo "##vso[task.setvariable variable=architecture]"
-          displayName: Initialize Test Variables
+  # - template: ../jobs/test-images-linux-client.yml
+  #   parameters:
+  #     name: PreBuildValidation
+  #     pool: ${{ parameters.linuxAmd64Pool }}
+  #     testJobTimeout: ${{ parameters.linuxAmdTestJobTimeout }}
+  #     preBuildValidation: true
+  #     internalProjectName: ${{ parameters.internalProjectName }}
+  #     customInitSteps:
+  #       - ${{ parameters.customTestInitSteps }}
+  #       # These variables are normally set by the matrix. Since this test job is not generated
+  #       # by a matrix, we need to set them manually. They can be set to empty values since their
+  #       # values aren't actually used for the pre-build tests.
+  #       - powershell: |
+  #           echo "##vso[task.setvariable variable=productVersion]"
+  #           echo "##vso[task.setvariable variable=imageBuilderPaths]"
+  #           echo "##vso[task.setvariable variable=osVersions]"
+  #           echo "##vso[task.setvariable variable=architecture]"
+  #         displayName: Initialize Test Variables
```

Comment:
Can we file an issue to remove this on the next patch tuesday?

Reply:
Would be nice to have a link to a tracking issue.

Reply:
Here's the issue: https://github.com/microsoft/dotnet-framework-docker/issues/1075